docs/serving/autoscaling.md

```diff
@@ -179,11 +182,26 @@ template.
 spec:
   template:
     metadata:
-      autoscaling.knative.dev/metric: concurrency
+      autoscaling.knative.dev/metric: cpu
       autoscaling.knative.dev/class: hpa.autoscaling.knative.dev
 ```
 
-## Additional resources
+# Configuring the PodAutoscaler custom resource
+
```

Comment:
Add a few sentences on **why** one might want to do that. Content might be:

- You want an autoscaler that better fits your workloads
- You want to use the metrics Knative provides with a different algorithm, that better suits your workloads
- You want to use entirely different metrics